cluster-autoscaler/processors/status/scale_up_status_processor.go

```diff
@@ -42,3 +42,10 @@ type ScaleUpStatusProcessor interface {
 func NewDefaultScaleUpStatusProcessor() ScaleUpStatusProcessor {
 	return &EventingScaleUpStatusProcessor{}
 }
+
+// NoOpScaleUpStatusProcessor is a ScaleUpStatusProcessor implementations useful for testing.
+type NoOpScaleUpStatusProcessor struct{}
+
+// Process processes the state of the cluster after a scale-up.
+func (p *NoOpScaleUpStatusProcessor) Process(context *context.AutoscalingContext, state *ScaleUpStatus) {
```

Comment:
s/state/status

Reply:
I somehow managed to exclude the fix I was intending to do... Done